docs/get-started/README.md

```diff
@@ -184,6 +184,10 @@ Please, be aware. All data will be deleted and we will not be able to restore it
 
 ![Delete project](./delete_prj.png)
 
+## Collaboration 
+With collaboration, you can share work with your team-mate. You can invite him/her to make setup, deployment, other administration work. It can speed up the development process and decrease overall delivery time. 
```

Comment:
@MishaJexia maybe change "You can invite him/her" to "You can invite them to make a setup, do a deployment and other administration work. It can speed up the development process and decrease the overall delivery time."